browser/brave_shields/brave_shields_web_contents_observer.cc

```diff
@@ -56,11 +58,12 @@ namespace {
 // Content Settings are only sent to the main frame currently.
 // Chrome may fix this at some point, but for now we do this as a work-around.
 // You can verify if this is fixed by running the following test:
-// npm run test -- brave_browser_tests --filter=BraveContentSettingsAgentImplBrowserTest.*  // NOLINT
-// Chrome seems to also have a bug with RenderFrameHostChanged not updating
-// the content settings so this is fixed here too. That case is covered in
-// tests by:
-// npm run test -- brave_browser_tests --filter=BraveContentSettingsAgentImplBrowserTest.*  // NOLINT
+// npm run test -- brave_browser_tests
+// --filter=BraveContentSettingsAgentImplBrowserTest.*  // NOLINT Chrome seems
+// to also have a bug with RenderFrameHostChanged not updating the content
+// settings so this is fixed here too. That case is covered in tests by: npm run
+// test -- brave_browser_tests
+// --filter=BraveContentSettingsAgentImplBrowserTest.*  // NOLINT
```

Comment:
Looks like the formatting on the `NOLINT` lines needs to be manually fixed

Reply:
I think it's better to just remove `// NOLINT` so I'm going to do that